groups/rds/outputs.tf

```diff
@@ -0,0 +1,7 @@
+output vpc_ids {
+  value = data.aws_vpc.placement.id
+}
+
+output subnet_ids {
+  value = local.placement_subnet_ids
+}
```

Comment:
Can we make sure we add newlines to the end of files as well please. This should be an editor setting